docs/source/usage/wm-widgets.rst

```diff
@@ -44,12 +44,31 @@ Add the following to your :file:`~/.config/qtile/config.py`:
        ),
    ]
 
+.. _bar-usage:
+
+bar
```

Comment:
@S0lll0s Of course, *not*. This is section name, not an identifier. Something like `LemonBoy’s bar` (if you use this better copy from here: I use U+2019 for apostrophe all over documentation if I write it myself. This variant usually looks better then `LemonBoy's bar` with U+0027).